SS_ALK.tpl

```diff
@@ -123,14 +123,14 @@ FUNCTION void Make_AgeLength_Key(const int s, const int subseas)
               }
 
               int ALK_phase = 0;
-              if (Grow_logN == 0)
-              {
-                int ALK_finder = (ALK_idx - 1) * gmorph + g;
-                ivector ALK_range_lo (0, nages);
-                ivector ALK_range_hi (0, nages);
-                ALK(ALK_idx, g) = calc_ALK(len_bins, ALK_range_lo, ALK_range_hi, use_Ave_Size_W, use_SD_Size);
-              }
-              else
+            //  if (Grow_logN == 0)
+            //  {
+            //    int ALK_finder = (ALK_idx - 1) * gmorph + g;
+            //    ivector ALK_range_lo (0, nages);
+            //    ivector ALK_range_hi (0, nages);
+            //    ALK(ALK_idx, g) = calc_ALK(len_bins, ALK_range_lo, ALK_range_hi, use_Ave_Size_W, use_SD_Size);
```

Comment:
Thanks for working on this @nschindler-noaa.
The model results changed by a lot according to the tests. Perhaps line 131, the call to `calc_ALK()` needs to be added back? It looks like that function doesn't actually make use of the `ALK_range_lo` and `ALK_range_hi` inputs, so maybe those should be removed from the function?

Reply:
I should have noted that the warnings generated by the github action are definitely reduced, but there still remain a few (pasted below from the artifact at the bottom of https://github.com/nmfs-stock-synthesis/stock-synthesis/actions/runs/4309790387) :
```
In file included from ss.cpp:7:
In file included from ss.cpp:7:
ss.cpp: In member function ‘void model_parameters::Make_AgeLength_Key(int, int)’:
ss.cpp:38123:19: warning: unused variable ‘ALK_phase’ [-Wunused-variable]
38123 |               int ALK_phase = 0;
      |                   ^~~~~~~~~
ss.cpp: In member function ‘dvar_matrix model_parameters::calc_ALK(const dvector&, const ivector&, const ivector&, const dvar_vector&, const dvar_vector&)’:
ss.cpp:38187:80: warning: unused parameter ‘ALK_range_lo’ [-Wunused-parameter]
38187 | dvar_matrix model_parameters::calc_ALK(const dvector& len_bins, const ivector& ALK_range_lo, const ivector& ALK_range_hi, const dvar_vector& mean_len_at_age, const dvar_vector& sd_len_at_age)
      |                                                                 ~~~~~~~~~~~~~~~^~~~~~~~~~~~
ss.cpp:38187:109: warning: unused parameter ‘ALK_range_hi’ [-Wunused-parameter]
38187 | dvar_matrix model_parameters::calc_ALK(const dvector& len_bins, const ivector& ALK_range_lo, const ivector& ALK_range_hi, const dvar_vector& mean_len_at_age, const dvar_vector& sd_len_at_age)
      |                                                                                              ~~~~~~~~~~~~~~~^~~~~~~~~~~~
```